src/jquery.fancytree.dnd5.js

```diff
@@ -502,13 +502,18 @@
 					break;
 			}
 
+			var dropMarkerRect;
+			if(!dndOpts.dropMarkerRoot.getBoundingClientRect)
+				dropMarkerRect = dndOpts.dropMarkerRoot.host.getBoundingClientRect();
+			else
+				dropMarkerRect = dndOpts.dropMarkerRoot.getBoundingClientRect();
 			pos = {
-				my: "left" + offsetString(markerOffsetX) + " center",
+				my: "left" + offsetString(markerOffsetX + dropMarkerRect.x) + " center",
```

Comment:
Can you give some explanations, what you are going to fix here?

Reply:
> Can you give some explanations, what you are going to fix here?

This was wrong ....
Sorry, had some strange thoughts